nixos/modules/services/system/nscd.nix

```diff
@@ -80,7 +88,10 @@ in
         serviceConfig =
```

Comment:
The comment above this should be modified/removed, because it won't be true anymore

Reply:
Thanks, I updated the comment